src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -90,6 +85,19 @@ private class SearchResult : INavigateToSearchResult
                     parts.Free();
                 }
             }
+
+            public string SecondarySort
```

Comment:
However, many of these changes are intended tohelp the core platform pattern matcher as well.  That will be rnuning in proc (possibly by many different components) so avoiding this is useful.